batch/batch/batch.py

```diff
@@ -305,7 +307,7 @@ async def unschedule_job(app, record):
             break
         try:
             async with aiohttp.ClientSession(
-                    raise_for_status=True, timeout=aiohttp.ClientTimeout(total=60)) as session:
+                    timeout=aiohttp.ClientTimeout(total=60)) as session:
```

Comment:
Why can you delete the raise_for_status here and the change directly below. They're not `client_session`